### PR TITLE
fix #1, problem with wire:model

### LINKED ISSUE #1: Currently, there's an issue when using `wire:model`
When using `x-autosize` together with `wire:model` the size of the `textarea` is not calculated correctly.

### PR DESCRIPTION
**Pull request by leobm**
I think I fixed the problem.

regards, 
Felix

### CODE REVIEW

**Comment by leobm** [comment]
The fix prevents that when the server model is updated, but not the textarea DOM in the browser, alpine should takes care of that. 

example:
```html

<div x-data="{message: ''}">
        <textarea x-autosize x-model="message" wire:model="message"></textarea>
        
        <!-- reflects server model -->
        <pre>{{$message}}</pre>
         
        <hr />

        <!-- reflects alpine model -->
        <pre x-text="message"></pre>
</div>
```

**Comment by marcreichel** [comment]
@leobm I can confirm this works! Thanks!

File: src/index.js [hunk 1]
```diff
@@ -1,11 +1,15 @@
...
+        if (el.hasAttribute('wire:model')) {
+            el.setAttribute('wire:ignore', '');
+        }
+
         const previousResizeValue = el.style.resize;
         el.style.resize = 'none';
 
         const previousMinHeight = el.style.minHeight;
         el.style.minHeight = el.getBoundingClientRect().height + 'px';
-
```
Comment (marcreichel) [inline src/index.js:8]: Please keep this line in the code 👍🏼

File: src/index.js [hunk 2]
```diff
@@ -1,11 +1,15 @@
 function Autosize(Alpine) {
     Alpine.directive('autosize', (el, {}, { cleanup }) => {
+        
+        if (el.hasAttribute('wire:model')) {
```
Comment (marcreichel) [inline src/index.js]:
```suggestion
        if (!el.hasAttribute('wire:ignore') && el.hasAttribute('wire:model')) {
```

Prevents an unnecessary DOM update (if `wire:ignore` is already set).

**Review by marcreichel — CHANGES_REQUESTED** [review]

**Review by marcreichel — APPROVED** [review]

**Pull request merged**